bin/runtime/src/elections/mod.rs

```diff
@@ -0,0 +1,316 @@
+use crate::{
+    AccountId, BlockNumber, EraIndex, Perquintill, Runtime, Session, SessionPeriod, SessionsPerEra,
+    Staking, Vec,
+};
+use pallet_elections::MembersPerSession;
+use primitives::{SessionIndex, TOKEN};
+
+fn blocks_to_produce_per_session() -> u32 {
+    SessionPeriod::get() / MembersPerSession::<Runtime>::get()
+}
+
+fn scale_total_exposure(total: u128) -> u128 {
+    // to cover minimal possible theoretical stake (25k) and avoid loss of accuracy we need to scale
+    100 * total / TOKEN
+}
+
+fn scaled_total_exposure(era: u32, validator: &AccountId) -> u128 {
+    let total = pallet_staking::ErasStakers::<Runtime>::get(era, validator).total;
+
+    scale_total_exposure(total)
+}
+
+fn validator_points_per_block(
+    total_exposure_in_tokens: u128,
+    sessions_per_era: u32,
+    blocks_to_produce_per_session: u32,
+) -> u32 {
+    (Perquintill::from_rational(1, (blocks_to_produce_per_session * sessions_per_era) as u64)
+        * total_exposure_in_tokens) as u32
+}
+
+fn reward_for_session_non_committee(committee: &[AccountId]) {
```

Comment:
Suggestion: it would be clearer if you would just use here a list of members not in committee instead or not use any parameter (similarly to other functions here).

Reply:
done